API.md

```diff
@@ -0,0 +1,552 @@
+# API
+
+* [Static Functions](#static-functions)
+  * [`create`](#create)
+* [Instance Methods](#instance-methods)
+  * [`start`](#start)
+  * [`stop`](#stop)
+  * [`dial`](#dial)
+  * [`dialProtocol`](#dialProtocol)
+  * [`hangUp`](#hangUp)
+  * [`handle`](#handle)
+  * [`unhandle`](#unhandle)
+  * [`peerRouting.findPeer`](#peerRouting.findPeer)
+  * [`contentRouting.findProviders`](#contentRouting.findProviders)
+  * [`contentRouting.provide`](#contentRouting.provide)
+  * [`_dht.put`](#_dht.put)
+  * [`_dht.get`](#_dht.get)
+  * [`_dht.getMany`](#_dht.getMany)
+  * [`pubsub.getPeersSubscribed`](#pubsub.getPeersSubscribed)
+  * [`pubsub.getTopics`](#pubsub.getTopics)
+  * [`pubsub.publish`](#pubsub.publish)
+  * [`pubsub.subscribe`](#pubsub.subscribe)
+
+## Static Functions
+
+### create
+
+Creates an instance of Libp2p.
+
+`create(options)`
+
+#### Parameters
+
+| Name | Type | Description |
+|------|------|-------------|
+| options | `Object` | libp2p options |
+
+#### Returns
+
+| Type | Description |
+|------|-------------|
+| `Promise<Libp2p>` | Promise resolves with the Libp2p instance |
+
+#### Example
+
+```js
+const Libp2p = require('libp2p')
+
+// specify options
+const options = {}
+
+// create libp2p
+const libp2p = await Libp2p.create(options)
+```
+
+Note: It is important pointing out that with `create`, the `PeerInfo` option is not required and will be generated if it is not provided.
+
+As an alternative, it is possible to create a Libp2p instance with the constructor:
```

Comment:
In js-ipfs we did this:

<img width="460" alt="Screenshot 2019-11-29 at 15 29 49" src="https://user-images.githubusercontent.com/152863/69878567-39704780-12bd-11ea-9269-6050854c54b0.png">

...then clicking to expand:

<img width="905" alt="Screenshot 2019-11-29 at 15 30 10" src="https://user-images.githubusercontent.com/152863/69878582-42f9af80-12bd-11ea-92a6-895eb5eedf3b.png">

This helps to reduce the amount of info the reader has to read and understand (whilst still keeping it available) for methods that are deprecated or not recommended.

Reply:
cool, I will copy this approach! Thanks 🙏